### PR TITLE
View and search locations

### PR DESCRIPTION
**Pull request by georgeschena**
This PR involves adding a searchable table of all the locations using GovWifi. This allows super admins to dynamically search through the table to find a desired location, the searching works either by location name or via postcode

Screenshot 2019-05-20 at 10 29 02

Searching via postcode: 
Screenshot 2019-05-20 at 10 30 27

### CODE REVIEW

File: app/views/admin/locations/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h1 class="govuk-heading-l">GovWifi locations </h1>
```
Comment (emileswarts) [inline app/views/admin/locations/index.html.erb]: extra space before closing </h1>

Comment (emileswarts) [reply app/views/admin/locations/index.html.erb]: h1

File: app/controllers/admin/locations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Admin::LocationsController < AdminController
+  def index
+    @locations = Location.all
```
Comment (emileswarts) [inline app/controllers/admin/locations_controller.rb]: Does calling .order without .all do the same thing?
I'm not sure, can you check?

Comment (georgeschena) [reply app/controllers/admin/locations_controller.rb]: Can confirm works the same. So removing `.all`

**Pull request merged**